main.tex

```diff
@@ -1,12 +1,10 @@
 \documentclass[oneside]{book}
+\usepackage{float}
 \input{preamble}
 \input{jupyter}
 
 \title{SP19 DL collaborative notes}
 \author{
-  The students of SP19 Deep Learning\\
-  Editor: Alfredo Canziani\\
-  NYU
```

Comment:
And we're at NYU.

Reply:
I didn't realize this was removed. Sorry for that. Will make the change